### DISCUSSION TITLE
Accent color on title bar appears really ugly. How to deal with it?

### DISCUSSION THREAD

**Discussion by fr1g** [lepoco/wpfui#690] [Q&A]
![Screenshot taken in sandbox](https://github.com/lepoco/wpfui/assets/53085628/5df4a2a0-0c1c-44d3-b033-b3698b86037a)
If the switch toggled on here, the app will appear like this on my machine:
![Ive hide some texts](https://github.com/lepoco/wpfui/assets/53085628/0dc14d54-b02b-47c9-a1ad-53c47b59d292)
 I'm new here in wpf developing, and before I start this I always using html, these stuff here made me felt funky. Kinda less freedom? lol. 
<br>But anyway, hope to know how do you deal with it. Except let user turn that switch off.

**Comment by JohnTasler** [comment]
I found this issue because I'm also looking into some accent-related things.

It's been several versions since your original post. Is the problem still occurring in 4.0.0.0? If so, please upload a small repro, in a ZIP or 7Z file.

**Comment by fr1g** [comment]
sorry recently I removed my environment developing WPF (VS), but I can try to make a repro via rider a bit later. Anyway it was happened when Windows enabled accent color on title bar of window. 发自我的 iPhone在 2025年2月14日
